component/common/src/main/java/org/wso2/carbon/extension/identity/emailotp/common/dto/ConfigsDTO.java

```diff
@@ -0,0 +1,154 @@
+
+        isEnabled = enabled;
+    }
+
+    public boolean isShowFailureReason() {
+
+        return showFailureReason;
+    }
+
+    public void setShowFailureReason(boolean showFailureReason) {
+
+        this.showFailureReason = showFailureReason;
+    }
+
+    public boolean isAlphaNumericOTP() {
+
+        return isAlphaNumericOTP;
+    }
+
+    public void setAlphaNumericOTP(boolean alphaNumericOTP) {
+
+        isAlphaNumericOTP = alphaNumericOTP;
+    }
+
+    public boolean isTriggerNotification() {
+
+        return triggerNotification;
+    }
+
+    public void setTriggerNotification(boolean triggerNotification) {
+
+        this.triggerNotification = triggerNotification;
+    }
+
+    public boolean isResendSameOtp() {
+
+        return resendSameOtp;
+    }
+
+    public void setResendSameOtp(boolean resendSameOtp) {
+
+        this.resendSameOtp = resendSameOtp;
+    }
+
+    public boolean isResendThrottlingEnabled() {
+
+        return resendThrottlingEnabled;
+    }
+
+    public void setResendThrottlingEnabled(boolean resendThrottlingEnabled) {
+
+        this.resendThrottlingEnabled = resendThrottlingEnabled;
+    }
+
+    public int getOtpLength() {
+
+        return otpLength;
+    }
+
+    public void setOtpLength(int otpLength) {
+
+        this.otpLength = otpLength;
+    }
+
+    public int getOtpValidityPeriod() {
+
+        return otpValidityPeriod;
+    }
+
+    public void setOtpValidityPeriod(int otpValidityPeriod) {
+
+        this.otpValidityPeriod = otpValidityPeriod;
+    }
+
+    public int getOtpRenewalInterval() {
+
+        return otpRenewalInterval;
+    }
+
+    public void setOtpRenewalInterval(int otpRenewalInterval) {
+
+        this.otpRenewalInterval = otpRenewalInterval;
+    }
+
+    public int getResendThrottleInterval() {
+
+        return resendThrottleInterval;
+    }
+
+    public void setResendThrottleInterval(int resendThrottleInterval) {
+
+        this.resendThrottleInterval = resendThrottleInterval;
+    }
+
+    @Override
+    public String toString() {
+
+        StringBuilder sb = new StringBuilder("ConfigsDTO {");
+        sb.append("\n\tisEnabled = ").append(isEnabled)
+                .append(",\n\tshowFailureReason = ").append(showFailureReason)
+                .append(",\n\tisAlphaNumericOTP = ").append(isAlphaNumericOTP)
+                .append(",\n\ttriggerNotification = ").append(triggerNotification)
+                .append(",\n\tresendSameOtp = ").append(resendSameOtp)
+                .append(",\n\tresendThrottlingEnabled = ").append(resendThrottlingEnabled)
+                .append(",\n\totpLength = ").append(otpLength)
+                .append(",\n\totpValidityPeriod = ").append(otpValidityPeriod)
+                .append(",\n\totpRenewalInterval = ").append(otpRenewalInterval)
+                .append(",\n\tresendThrottleInterval = ").append(resendThrottleInterval)
+                .append("\n}");
+        return sb.toString();
```

Comment:
Can't we make use of `Gson().toJson()` in places this is called?

Reply:
This is `toString()` method no need to convert it to `JSON` object.